### PR TITLE
[10.0][FIX] account_financial_report_qweb: Add onchange_date_from

### PR DESCRIPTION
**Pull request by luismontalba**
Add onchange method to fill the first fiscal year date avoiding the mandatory use of a date range.

cc @Tecnativa

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]
Please review @cubells @rafaelbn

**Comment by sbidoul** [comment]
fy_start_date should be a computed field?

**Comment by pedrobaeza** [comment]
Yeah, indeed we can put it as computed and check date range or date start. What do you prefer?

**Comment by alexis-via** [comment]
I made a proposal to simplify the FIX https://github.com/Tecnativa/account-financial-reporting/pull/1/files but I read stephan's comment only now... I think it's better to make it a compute field. I'll update my PR.

**Comment by alexis-via** [comment]
I have updated my PR https://github.com/Tecnativa/account-financial-reporting/pull/1  This PR converts fy_start_date to a computed field. Let's merge my PR here and then merge this PR.

File: account_financial_report_qweb/wizard/general_ledger_wizard.py [hunk 1]
```diff
@@ -59,6 +61,14 @@ class GeneralLedgerReportWizard(models.TransientModel):
         string='Not only one unaffected earnings account'
     )
 
+    @api.depends('date_from')
+    def compute_fy_start_date(self):
```
Comment (sbidoul) [inline account_financial_report_qweb/wizard/general_ledger_wizard.py]: underscore _compute for security

File: account_financial_report_qweb/README.rst [hunk 2]
```diff
@@ -51,6 +51,7 @@ Contributors
 * Lorenzo Battistini <lorenzo.battistini@agilebg.com>
 * Julien Coux <julien.coux@camptocamp.com>
 * Akim Juillerat <akim.juillerat@camptocamp.com>
+* Luis M. Ontalba luis.martinez@tecnativa.com>
```
Comment (sbidoul) [inline account_financial_report_qweb/README.rst]: add alexis

**Comment by pedrobaeza** [comment]
Making the changes...

File: account_financial_report_qweb/wizard/trial_balance_wizard.py [hunk 3]
```diff
@@ -53,6 +55,14 @@ class TrialBalanceReportWizard(models.TransientModel):
         string='Not only one unaffected earnings account'
     )
 
+    @api.depends('date_from')
+    def compute_fy_start_date(self):
```
Comment (sbidoul) [inline account_financial_report_qweb/wizard/trial_balance_wizard.py]: _compute

**Review by alexis-via — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@sbidoul give your final blessing.

**Comment by alexis-via** [comment]
ok, great work, let's merge now.

**Comment by pedrobaeza** [comment]
OK, I'll wait Travis to finish for avoiding any dumb error, but merging then.

**Review by sbidoul — APPROVED** [review]

**Pull request merged**